internal/util/update/update_test.go

```diff
@@ -728,7 +728,7 @@ func TestCommand_Run_subpackages(t *testing.T) {
 			},
 			expectedResults: []resultForStrategy{
 				{
-					strategies: []StrategyType{KResourceMerge},
+					strategies: []StrategyType{KResourceMerge, FastForward},
```

Comment:
Not sure if this case is covered. What if Kptfile is deleted from local in one of the subpackages. Is it considered as altering the package during fast-forward? How are the upstream changes merged with local?